ops/framework.py

```diff
@@ -212,6 +215,18 @@ def __init__(self, parent, key):
 
         # TODO Detect conflicting handles here.
 
+    @property
+    def model(self):
+        return self.framework.model
```

Comment:
Since they're not expected to change, we could do actual copy / references rather than properties.  I also think that it might be worth including some of the second-order attributes such as `self.app = self.framework.model.app` and `self.unit = self.framework.model.unit` since they're commonly used and do make some logical sense when the charm refers to them as "self.app` but I didn't want to go to far at once with this PR.